sfdc37/tests/test.bal

```diff
@@ -42,7 +49,7 @@ function testGetAvailableApiVersions() {
     json|SalesforceConnectorError jsonRes = salesforceClient->getAvailableApiVersions();
 
     if (jsonRes is json) {
-        test:assertNotEquals(jsonRes, null, msg = "Found null JSON response!");
+        test:assertNotEquals(jsonRes, (), msg = "Found null JSON response!");
```

Comment:
Do we need to specifically check this? We are casting to a `json[]` in the next line which could fail anyway if it is some other type. Can this actually be `()`?